modules/core/keeper/msg_server.go

```diff
@@ -100,6 +100,11 @@ func (k Keeper) SubmitMisbehaviour(goCtx context.Context, msg *clienttypes.MsgSu
 	return &clienttypes.MsgSubmitMisbehaviourResponse{}, nil
 }
 
+// RecoverClient defines a rpc handler method for MsgRecoverClient.
+func (k Keeper) RecoverClient(goCtx context.Context, msg *clienttypes.MsgRecoverClient) (*clienttypes.MsgRecoverClientResponse, error) { //nolint:revive
```

Comment:
```suggestion
func (Keeper) RecoverClient(goCtx context.Context, msg *clienttypes.MsgRecoverClient) (*clienttypes.MsgRecoverClientResponse, error) {
```

Reply:
yea, I'll remove the nolint directive when adding the implementation for RecoverClient